+adi/+FMComms5/Rx.m

```diff
@@ -0,0 +1,337 @@
+classdef Rx < adi.FMComms5.Base & adi.AD9361.Rx ...
+        & adi.FMComms5.TuneAGCFMComms5ChipB
+    % adi.FMComms5.Rx Receive data from the FMComms5 transceiver
+    %   The adi.FMComms5.Rx System object is a signal source that can receive
+    %   complex data from the FMComms5.
+    %
+    %   rx = adi.FMComms5.Rx;
+    %   rx = adi.FMComms5.Rx('uri','192.168.2.1');
+    %
+    %   <a href="http://www.analog.com/media/en/technical-documentation/data-sheets/AD9361.pdf">AD9361 Datasheet</a>
+    %
+    %   See also adi.FMComms2.Rx, adi.FMComms3.Rx, adi.AD9361.Rx
+    
+    properties
+        %CenterFrequency Center Frequency
```

Comment:
Update all 3 parameters to include "Chip B" in docstrings. So CenterFrequency Center Frequency -> CenterFrequencyChipB Center Frequency Chip B